extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/BackChannelLogoutHandler.java

```diff
@@ -59,6 +59,9 @@ public void handle(RoutingContext context) {
                 LOG.debugf(
                         "Tenant configuration for the tenant %s is not available or does not match the backchannel logout path",
                         oidcTenantConfig.getTenantId().get());
+                context.response().setStatusCode(400);
```

Comment:
I wish we could include the reason in the response. Returning a client error without any reason will make it harder to debug situations where this happens

Reply:
@gastaldi I see, but note this one will be returned to Keycloak directly, not to Quarkus user; besides this is a case where the fact Quarkus can't find the matching configuration is probably not something we'd like to say to the outside world, logging it for the users to trace seems reasonable. Perhaps we can make this one an ERROR-level message though, what do you think ?

Reply:
Not sure if that's a good idea as clients could easily fill the logs with this information because the request is invalid

Reply:
@gastaldi I changed the log level to `error`, hope it is better